src/browser.ts

```diff
@@ -11,7 +11,7 @@ export default function(history: History = createBrowserHistory()) {
   // there's only `view` which is the whole `pathname` from history perspective
 
   function syncToBrowser(s: BrowserState, push: boolean = true): void {
-    const pathname = `/${s.view}`;
+    const pathname = `/${s.view || ''}`;
```

Comment:
then update BrowserState to have `view` optional? (right now it is `view: string`)